<launch>
<rosparam param="CHI">     1.75 </rosparam>
<rosparam param="Inv_RATIO">     38.7 </rosparam>

<node pkg="tf" type="static_transform_publisher" name="toworld" args="0.16821159 -0.6409790 -0.34234114 -0.0079871 0.018178 -0.62607 0.77952 odom world 100" />
<!-- node pkg="tf" type="static_transform_publisher" name="toworld" args="-0.6683338 -0.0377706 0.32901361757 0.0079871 -0.018178 0.62607 0.77952 world odom 100" -->

<!-- Run  the node that reads the wheels velocities from the bag and publishes the Twist message-->
<node pkg = "project1" name = "velpubsub" type = "velpubsub" output="screen" />

<!-- Run the pose node. Theta must be in radians -->
<node pkg = "project1" name = "odompubsub" type = "odompubsub" output="screen">
    <rosparam param="x_init">     0.0 </rosparam>
    <rosparam param="y_init">     0.0 </rosparam>
    <rosparam param="theta_init"> 0.0 </rosparam>
</node>

<node pkg = "project1" name = "residuals" type = "residuals" output="screen" />

</launch>